rudder-lang/Cargo.toml

```diff
@@ -32,6 +32,5 @@ proptest = "0.9"
 test-case = "0.3"
 
 [build-dependencies]
-ron = "0.5"
 serde = { version = "1", features = ["derive"] }
```

Comment:
do we still have build-dependencies actually?

Reply:
no but i'll remove it from my pr, more related